src/NetscriptFunctions.ts

```diff
@@ -1836,6 +1838,17 @@ export const ns: InternalAPI<NSFull> = {
   printRaw: (ctx) => (value) => {
     ctx.workerScript.print(wrapUserNode(value));
   },
+  importFrom: (ctx) => (value) => {
```

Comment:
The MDN for import() is very clear that it will *not* synchronously throw any errors (they'll be part of promise rejection). I think we should mirror import() as closely as possible. That means we should make this function async, so anything it throws (or compile throws) becomes part of the promise return.